src/component/mxgraph/MxGraphConfigurator.ts

```diff
@@ -25,6 +40,46 @@ export default class MxGraphConfigurator {
     this.configureExclusiveGatewayStyle();
   }
 
+  private initMxGraphPrototype(): void {
+    this.mxGraph.prototype.edgeLabelsMovable = false;
+    this.mxGraph.prototype.cellsLocked = true;
+    this.mxGraph.prototype.cellsSelectable = false;
+    const isFF = this.mxClient.IS_FF;
+
+    // this change is needed for adding the custom attributes that permits identification of the BPMN elements
+    this.mxShape.prototype.createSvgCanvas = function() {
+      const mxSvgCanvas2D: any = MxGraphFactoryService.getMxGraphProperty('mxSvgCanvas2D');
```

Comment:
Maybe it would be better to externalize this function in other function, in order to have few lines in initMxGraphPrototype() ;)